applications/StructuralMechanicsApplication/tests/truss_test/linear_3D2NTruss_plastic_compression_test_parameters.json

```diff
@@ -0,0 +1,143 @@
+{
+    "problem_data"             : {
+        "problem_name"    : "linear_3D2NTruss_compression_plasticity_test",
+        "model_part_name" : "Structure",
+        "domain_size"     : 3,
+        "parallel_type"   : "OpenMP",
+        "time_step"       : 0.5,
+        "start_time"      : 0.0,
+        "end_time"        : 210,

```

Comment:
how long does this simulation run?

Reply:
like 1s

Reply:
maybe 2 :D

Reply:
but here it is important to capture some hysteresis cycles in the big test

Reply:
Fine, should be in the ValidationSuite then